### DISCUSSION TITLE
Trying to deploy with font awesome pro

### DISCUSSION THREAD

**Discussion by jamitz440** [coollabsio/coolify#2692] [Q&A]
I have an .npmrc file set up like so:

```
@awesome.me:registry=https://npm.fontawesome.com/
@fortawesome:registry=https://npm.fontawesome.com/
//npm.fontawesome.com/:_authToken=${NPM_AUTH}
```

and a NPM_AUTH environment variable. 

But it fails on build. I am quite new to self hosted deployments and cant seem to figure this one out. 

anyone know a possible solution for this?

**Comment by peaklabs-dev** [comment]
@jamitz440 What exactly are you trying to deploy, can you maybe give me a repository URL?
